### PR TITLE
Homepage WIP

### PR DESCRIPTION
**Pull request by amygtownsend**
This PR adds assets and additional styles to homepage, creates a context for json data and applies it, creates separate Home and Excerpts components and adds a custom Tailwind variant.
Fed Blog Homepage

### CODE REVIEW

File: src/components/Home.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          <img src={arrowLeft} alt="previous blog posts" className="mr-8" />
+          <button className="btn">Prev</button>
+        </div>
+        <div className="flex items-center">
+          <button className="btn">Next</button>
+          <img src={arrowRight} alt="next blog posts" className="ml-8" />
+        </div>
+      </div>
+    </main>
+  </div>
```
Comment (benjtinsley) [inline src/components/Home.js]: maybe you are heading this way already, but id break these into smaller components to keep this from getting too hard to maintain or understand. 

im thinking a header, selector, post list and pagination would all be good candidates

File: src/components/Excerpts.js [hunk 2]
```diff
@@ -35,38 +29,52 @@ const Excerpts = () => {
...
+
     const cardClasses = classNames(
-      'group flex items-start p-30 my-30 hover:text-white rounded-8',
+      'group flex items-start p-30 my-30 hover:text-white focus:text-white rounded-8',
       {
         'hover:bg-green-200': index % 2 === 0,
-        'hover:bg-blue-200': index % 2 !== 0
+        'focus:bg-green-200': index % 2 === 0,
+        'hover:bg-blue-200': index % 2 !== 0,
+        'focus:bg-blue-200': index % 2 !== 0
```
Comment (leobauza) [inline src/components/Excerpts.js]: these can be in 1 line:

`'hover:bg-blue-200 focus:bg-blue-200': index % 2 !== 0`

File: src/components/Excerpts.js [hunk 3]
```diff
@@ -35,38 +29,52 @@ const Excerpts = () => {
...
+    const day = d.getDay()
+    const year = d.getFullYear()
+
     const cardClasses = classNames(
-      'group flex items-start p-30 my-30 hover:text-white rounded-8',
+      'group flex items-start p-30 my-30 hover:text-white focus:text-white rounded-8',
       {
         'hover:bg-green-200': index % 2 === 0,
-        'hover:bg-blue-200': index % 2 !== 0
+        'focus:bg-green-200': index % 2 === 0,
```
Comment (leobauza) [inline src/components/Excerpts.js]: these can be in 1 line:

`'hover:bg-green-200 focus:bg-green-200': index % 2 === 0,`

File: src/components/Home.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+          <option value="all">All</option>
+          <option value="category1">Category 1</option>
+          <option value="category2">Category 2</option>
+          <option value="category3">Category 3</option>
+          <option value="category4">Category 4</option>
+        </select>
+      </div>
+      <hr className="border-gray-100 border m-0 mx-30" />
+      <div className="flex-col">
+        <DataContext.Consumer>
```
Comment (leobauza) [inline src/components/Home.js]: usually things are more readable if you have this context inside the `Excerpts` component that uses it

File: src/css/components/buttons.css [hunk 5]
```diff
@@ -1,4 +1,7 @@
 .btn {
-  @apply text-white;
-  @apply bg-blue-200;
+  @apply text-purple;
+  @apply text-50;
+  @apply font-sans;
+  @apply font-extralight;
+  @apply uppercase;
```
Comment (leobauza) [inline src/css/components/buttons.css]: 🔤

File: tailwind.config.js [hunk 6]
```diff
@@ -60,6 +62,7 @@ module.exports = {
       right: 'right',
       'right-bottom': 'right bottom',
       'right-top': 'right top',
+      'right-8': 'right .5rem top 50%',
```
Comment (leobauza) [inline tailwind.config.js]: you have a `rem` function so `right ${rem(8)} top 50%` I think will work

File: tailwind.config.js [hunk 7]
```diff
@@ -340,5 +344,13 @@ module.exports = {
...
-  plugins: []
+  plugins: [
+    function({ addVariant, e }) {
+      addVariant('group-focus', ({ modifySelectors, separator }) => {
+        modifySelectors(({ className }) => {
+          return `.group:focus .${e(`group-focus${separator}${className}`)}`
+        })
+      })
+    }
+  ]
```
Comment (leobauza) [inline tailwind.config.js]: v-cool

**Pull request merged**